docs/semgrep-ci.md

```diff
@@ -1,180 +1,283 @@
 ---
 append_help_link: true
+meta_description: |-
+  Semgrep CI is a specialized Docker image for running Semgrep in CI environments. It can also be used stand-alone or connected with Semgrep App for centralized rule and findings management. Rapidly scan every commit and block new bugs with Semgrep CI.
```

Comment:
```suggestion
  Semgrep CI is a specialized Docker image for running Semgrep in CI environments. It can either be used stand-alone or connected with Semgrep App for centralized rule and findings management. Rapidly scan every commit and block new bugs with Semgrep CI.
```